.github/workflows/master-e2e.yaml

```diff
@@ -300,6 +300,17 @@ jobs:
       - name: List installed nodes
         if: inputs.test_type == 'cli'
         run: sudo virsh list
+      - name: Store logs
+        if: always()
+        run: |
+          cd tests && make e2e-get-logs
+      - name: Upload downstream cluster logs
```

Comment:
This mentions downstream, but we are getting the upstream logs?

Reply:
yep exactly, actually I need to implement upstream logs to debug other issues in the CI.
I will add downstream logs later, I'm not able to quickly code scp go client etc...